eve/main.yml

```diff
@@ -68,7 +68,7 @@ models:
   - SetProperties: &set_operator_properties
       properties:
         OPERATOR_IMAGE_NAME: registry.scality.com/sf-eng/zenko-operator
```

Comment:
we should switch to using a different namespace for zenko operator, maybe for the next release @tcarmet

Reply:
after talking to Alex C there might be an issue given zenko operator is private while zenko is public, so where to place the image is an issue